test/Feature/Atomics.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: %llvmas %s -o=%t.bc
+; RUN: rm -rf %t.klee-out
```

Comment:
Please replace this test with C code. This is easier to maintain and the compiler does the hard work.
You can get inspired here: `https://github.com/llvm-mirror/clang/blob/master/test/CodeGen/atomic-ops.c`.

Beside that, please add a simple test for each operation.

Reply:
@kren1 One test file with multiple checks for the different operations.